src/components/common/FormInputError.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+
+import { FormMessage } from 'components/common/FormMessage'
+
+const Wrapper = styled.div<{ $margin?: string; $padding?: string }>`
+  margin: ${({ $margin = '0' }): string => $margin};
+  padding: ${({ $padding = '0' }): string => $padding};
+`
+
+export interface Props {
+  errorMessage?: string
```

Comment:
You can add multiline doc comments that get shown in storybook docs tab
like in EtherscanLink component

Reply:
Yep, although I think for FormInput it's ok to leave it undocumented.
We can try to document a bit more the components that are important. This one in particular, is self explanatory (it has just one field)